packages/core/core/src/Parcel.js

```diff
@@ -176,8 +176,8 @@ export default class Parcel {
     this.#initialized = false;
 
     await Promise.all([
-      this.#disposable.dispose(),
       await this.#requestTracker.writeToCache(),
+      this.#disposable.dispose(),
```

Comment:
We can't dispose the reporter yet because we're sending events in the `writeToCache` function now so I had to change the order here

Reply:
Isn't
```js
await Promise.all([
      await this.#requestTracker.writeToCache(), // this is just undefiend
      this.#disposable.dispose() // actually a promise
])
```
the same as
```js
await this.#requestTracker.writeToCache();
await this.#disposable.dispose();
```
anyway?